libs/common/include/helpers/mathFuncs.h

```diff
@@ -79,4 +81,18 @@ constexpr T inverseLerp(const T startVal, const T endVal, const T value) noexcep
 {
     return (value - startVal) / (endVal - startVal);
 }
+
+// TODO can be constexpr in C++20
+/// Arithmetically round floating point values to integers
+template<typename IntType, typename FloatType,
+         std::enable_if_t<std::is_integral<IntType>::value && std::is_floating_point<FloatType>::value, int> = 0>
+inline IntType iround(const FloatType val) noexcept
```

Comment:
Sorry missed that: The `inline` is redundant for template & constexpr functions. IIRC we follow this consistently so I don't want to introduce a different style.
```suggestion
IntType iround(const FloatType val) noexcept
```